resources/fetchers/file_system_fetcher_test.go

```diff
@@ -19,6 +19,9 @@ package fetchers
 
```

Comment:
Most definitely this test file has to be table-driven to reduce the repetition of code.